core/src/main/java/org/opensearch/sql/analysis/SelectExpressionAnalyzer.java

```diff
@@ -68,50 +65,44 @@ public List<NamedExpression> visitAlias(Alias node, AnalysisContext context) {
     }
 
     Expression expr = referenceIfSymbolDefined(node, context);
-    return Collections.singletonList(DSL.named(
-        unqualifiedNameIfFieldOnly(node, context),
-        expr,
-        node.getAlias()));
+    return Collections.singletonList(
+        DSL.named(unqualifiedNameIfFieldOnly(node, context), expr, node.getAlias()));
   }
 
   /**
-   * The Alias could be
-   * 1. SELECT name, AVG(age) FROM s BY name ->
-   * Project(Alias("name", expr), Alias("AVG(age)", aggExpr))
-   * Agg(Alias("AVG(age)", aggExpr))
-   * 2. SELECT length(name), AVG(age) FROM s BY length(name)
-   * Project(Alias("name", expr), Alias("AVG(age)", aggExpr))
-   * Agg(Alias("AVG(age)", aggExpr))
-   * 3. SELECT length(name) as l, AVG(age) FROM s BY l
-   * Project(Alias("name", expr, l), Alias("AVG(age)", aggExpr))
-   * Agg(Alias("AVG(age)", aggExpr), Alias("length(name)", groupExpr))
+   * The Alias could be 1. SELECT name, AVG(age) FROM s BY name -> Project(Alias("name", expr),
```

Comment:
This doesn't look great. It gets harder to read the bullet points. Is there anything we can do to change to allow old format?

Reply:
Thanks! fixed in [d04dd9f](https://github.com/Bit-Quill/opensearch-project-sql/pull/312/commits/d04dd9f04d966cfc1c0e047cccb8bab3e6c386ba)